steamclog/src/main/java/com/steamclock/steamclog/Destinations.kt

```diff
@@ -65,7 +64,20 @@ internal class SentryDestination : Timber.Tree() {
 
             // Always use the message as our error report, as this gives us way more contextual
             // info about the problem than the name of the Exception.
-            Sentry.captureMessage(originalMessage, SentryLevel.ERROR)
+            val sentryEvent = SentryEvent(originalThrowable).apply {
+                level = SentryLevel.ERROR
+                setMessage(Message().apply {
+                    setMessage(originalMessage)
+                    formatted = originalMessage
+                })
+                wrapper?.extraInfo?.let { extras = it }
+            }
+
+            val hintWithAttachment = wrapper?.logAttachmentUrl?.let {logFile ->
+                Hint.withAttachment(Attachment(logFile.absolutePath))
+            }
+
+            Sentry.captureEvent(sentryEvent, hintWithAttachment)
```

Comment:
More for my information, but since the ScopeCallback also has an `addAttachment()`, I was curious to know whether was this approach turned out to be easier to work with than `Sentry.captureEvent(scope, scopeCallback)`

Reply:
I was going by the Sentry documentation for adding an attachments, and it seemed easier to pass it in directly than to have to manipulate the scope (one parameter vs. a few lines to setup and attach to the scope):
https://docs.sentry.io/platforms/android/enriching-events/attachments/